scripts/misc.py

```diff
@@ -2,7 +2,10 @@
 A collection of commonly-used functions, could be cross-projects
 """
 import re
+import logging
 
+logger = logging.getLogger(__name__)
```

Comment:
It's worth to get instance of logger from utils module

Reply:
done in b9337f6